src/calculator.py

```diff
@@ -0,0 +1,6 @@
+a=int(input("Input first number: "))
+b=int(input("Input second number: "))
+print("Sum: "+str(a+b))
+print("Difference:"+str(a-b))
+print("Product: "+str(a*b))
+print("Division: "+str(a/b))
```

Comment:
L;interpolazione delle stringhe in python puo' essere gestita eleganemente in vari modi:

```py
# fstring
print(f"Sum: {a + b}")
# concatenazione classica
print("Sum: " + str(a + b))
# sfruttando i parametri di print
print("Sum:", a + b)
```